internal/dispatch/singleflight/singleflight_test.go

```diff
@@ -255,7 +255,7 @@ func assertCounterWithLabel(t *testing.T, gatherer prometheus.Gatherer, expected
 func bloomFilterForRequest(t *testing.T, keyHandler *keys.DirectKeyHandler, req *v1.DispatchCheckRequest) []byte {
 	t.Helper()
 
-	bloomFilter := bloom.NewWithEstimates(defaultBloomFilterSize, 0.01)
+	bloomFilter := bloom.NewWithEstimates(defaultBloomFilterSize, 0.001)
```

Comment:
use `defaultFalsePositiveRate` here?

Reply:
I don't think it's necessary, and it would import the `pkg/graph` package here, and it would need to be exported.

This package does not care about the size nor the probability of the bloom filter - it's specified externally by whoever initializes sets the `ResolverMeta`. The only contract is that it needs to use the same library.

So from the perspective of the tests, this could be any probability (we probably want to have it low so it does not flake).